src/templates/PoolV2/Hero/ShareImageModal/index.tsx

```diff
@@ -56,11 +63,20 @@ const ShareImageModal = ({
     }
   }
 
-  function handleShareClick() {
-    setUrl(`https://app.kassandra.finance/shared/${v4()}-${poolId}`)
+  async function handleShareClick() {
```

Comment:
Essa função não precisa ser async

```suggestion
  function handleShareClick() {
```